test/Microsoft.IdentityModel.Tokens.Tests/RsaCryptoServiceProviderProxyTests.cs

```diff
@@ -356,10 +356,10 @@ public static TheoryData<RSACryptoServiceProviderProxyTheoryData> RSAVerifyDataT
                 var rsaCsp = new RSACryptoServiceProvider();
                 rsaCsp.ImportParameters(KeyingMaterial.RsaParameters_2048);
 #else
-                var rsaCsp = KeyingMaterial.DefaultCert_2048.PrivateKey as RSACryptoServiceProvider;
+                var rsaCsp = KeyingMaterial.CertSelfSigned1024_SHA256.PrivateKey as RSACryptoServiceProvider;
```

Comment:
Is this cert working? If I am not mistaken this is an existing cert that we tried, should it be CertSelfSigned2048_SHA256_2?

Reply:
This cert works here.

Reply:
This test is only checking parameters, doesn't verify. Thanks @ciaozhang for clarifying.